packages/async-queue/README.md

```diff
@@ -58,3 +95,7 @@ Thank you to all the people who already contributed to Sapphire!
 </a>
 
 [contributing]: https://github.com/sapphiredev/.github/blob/main/.github/CONTRIBUTING.md
+
+```
+
+```
```

Comment:
huh?

Reply:
I'm just as confused as you are, idk where that comes from, I didn't write it